org.eclipse.scout.rt.app/src/main/java/org/eclipse/scout/rt/app/ApplicationProperties.java

```diff
@@ -209,6 +209,25 @@ protected SameSite parse(String value) {
     }
   }
 
+  public static class ScoutApplicationSessionCookieConfigPartitionedProperty extends AbstractBooleanConfigProperty {
+
+    @Override
+    public String getKey() {
+      return "scout.app.sessionCookieConfigPartitioned";
+    }
+
+    @Override
+    public Boolean getDefaultValue() {
+      return false;
+    }
+
+    @Override
+    public String description() {
+      return "Specifies whether any session tracking cookies created by the web application will be marked as partitioned. "
```

Comment:
I would rephrase it like so: Specifies whether the HTTP session cookie created  by the web... The term tracking is confusing. Please also adjust the description for scout.app.sessionCookieConfigSecure accordingly and remove the word 'tracking'